### PR TITLE
feat(client): thread skeleton loading component

### PR DESCRIPTION
**Pull request by vuhuucuong**
fixes: #19

### CODE REVIEW

File: client/components/ThreadSkeletonLoading/index.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+`;
+
+const createLoadingKeyframes = (width: number) => keyframes`
+    from {
+      background-position: -${width * 1.5}px 0;
+    }
+    to {
+      background-position: ${width}px 0;
+    }
+`;
```
Comment (zlatanpham) [inline client/components/ThreadSkeletonLoading/index.tsx]: There is a fixed number for the animation: 

```jsx
export const flash = keyframes`
  0% {
    background-position: -200px 0;
  }
  100% {
    background-position: calc(200px + 100%) 0;
  }
`;
```

File: client/components/ThreadSkeletonLoading/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+const TextContainer = styled.div`
+  padding-left: 10px;
+`;
+
+const TextLoading = styled.div<{ width: number; height: number }>`
+  ${props => createLoadingStyles(props.width, props.height)}
+  animation-name: ${props => createLoadingKeyframes(props.width)};
+  margin-bottom: 10px;
+`;
```
Comment (zlatanpham) [inline client/components/ThreadSkeletonLoading/index.tsx]: This somehow can be simplified to 

```jsx
const LoaderBlock = styled.div<{width: string, height: string, radius?: string}>`
  width: ${props => props.width};
  height: ${props => props.height};
  border-radius: ${props => props.radius || '0px'};
  background-color: #edf2f7;
  background: linear-gradient(to right, #edf2f7 0%, #cbd5e0 17%, #edf2f7 33%);
  background-size: 200px 100%;
  animation-duration: 1s;
  animation-delay: -0.5s;
  animation-iteration-count: infinite;
  animation-fill-mode: forwards;
`

// I don't think we need pass any props
const ThreadSkeletonLoading = () => {
	return (
		<div className="flex">
			<LoaderBlock className="flex-none mr-2 rounded-full" width="50px" height="50px"/>
			<div>
				<LoaderBlock className="mb-1" width="90%" height="8px"/>
				<LoaderBlock className="mb-1" width="80%" height="8px"/>
				<LoaderBlock className="mb-2" width="70%" height="8px"/>			
                ...
			</div>
		</div>
	)
}
```

**Comment by zlatanpham** [comment]
Hey @vuhuucuong, can you remove `client/yarn.lock` as it shouldn't be there. I just wonder how you run the project or storybook? If you would get started at the root, the `client/yarn.lock` should have not been generated.

**Comment by vuhuucuong** [comment]
> Hey @vuhuucuong, can you remove `client/yarn.lock` as it shouldn't be there. I just wonder how you run the project or storybook? If you would get started at the root, the `client/yarn.lock` should have not been generated.

I removed

**Pull request merged**